script.nhlscores/resources/settings.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
can you update this to the new settings format? There's an addon in the forum to do it automatically on your behalf